scripts/test/Muon/quick_edit/duel_quick_edit_test.py

```diff
@@ -0,0 +1,44 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+
+from unittest import mock
+
+from Muon.GUI.Common.plot_widget.quick_edit.quick_edit_view import QuickEditView
+from Muon.GUI.Common.plot_widget.quick_edit.quick_edit_presenter import DuelQuickEditPresenter
+from Muon.GUI.Common.contexts.plotting_context import PlottingContext
+
+from mantid.simpleapi import AnalysisDataService
+from mantidqt.utils.qt.testing import start_qapplication
+
+
+def plot_at_index(index):
+    return "plot "+str(index)
+
+
+@start_qapplication
+class DuelQuickEditTest(unittest.TestCase):
+
+    def setUp(self):
+        self.view = QuickEditView(None, None)
```

Comment:
Should this just be a mock type? We shouldn't have an actual Qt view in a test as it can lead to flakey test instances

Reply:
```suggestion
        self.view = mock.Mock(spec=QuickEditView)
```